### PR TITLE
Changes required for Composer tab

### PR DESCRIPTION
**Pull request by Graeme-Miller**

### CODE REVIEW

File: jsgui/src/main/webapp/index.html [hunk 1]
```diff
@@ -48,6 +51,7 @@
             <div class="nav-collapse menubar-top">
                 <ul class="nav">
                     <li><a href="#/v1/home" class="nav1 nav1_home">Home</a></li>
+                    <li><a href="#v1/editor/" class="nav1 nav1_editor">Composer</a></li>
```
Comment (grkvlt) [inline jsgui/src/main/webapp/index.html]: Why no initial `/` like the others?

Comment (neykov) [inline jsgui/src/main/webapp/index.html]: Leading slashes have been removed in brooklyn's html, better follow here for all menu links.

File: jsgui/src/main/webapp/index.html [hunk 2]
```diff
@@ -48,6 +51,7 @@
             <div class="nav-collapse menubar-top">
```
Comment (neykov) [inline jsgui/src/main/webapp/index.html]: Believe `<a class="btn btn-navbar" data-toggle="collapse" data-target=".nav-collapse">...<a class="brand"...>` above is no longer needed as well.

File: jsgui/src/main/webapp/index.html [hunk 3]
```diff
@@ -18,6 +18,9 @@
 
     <!-- Clocker Branding. -->
     <link rel="stylesheet" href="/assets/css/clocker.css">
```
Comment (neykov) [inline jsgui/src/main/webapp/index.html]: `brooklyn.css` contains some changes for codemirror and misc, believe they need to be moved over.

Comment (neykov) [inline jsgui/src/main/webapp/index.html]: Missing `<link rel="stylesheet" href="/assets/css/show-hint.css">`

File: jsgui/src/main/webapp/index.html (hunk 2 above)
Comment (neykov) [inline jsgui/src/main/webapp/index.html]: `Log out` link missing.

**Comment by grkvlt** [comment]
Thanks @neykov that's helpful. @Graeme-Miller can you take a look, please, as I would like to merge this.

File: jsgui/src/main/webapp/index.html (hunk 1 above)
Comment (grkvlt) [inline jsgui/src/main/webapp/index.html]: @neykov @Graeme-Miller so _all_ the links here should be changed to look like the editor one?

**Comment by Graeme-Miller** [comment]
Fixes made for comments. Also noticed swagger UI wasn't working, so have fixed that also

**Comment by grkvlt** [comment]
@Graeme-Miller looks good, merging

**Pull request merged**